plugins/aea-ledger-ethereum-hwi/setup.py

```diff
@@ -42,8 +42,8 @@
         "web3>=6.0.0,<7",
         "ipfshttpclient==0.8.0a2",
         "eth-account>=0.8.0,<0.9.0",
-        "open-aea-ledger-ethereum~=1.51.0",
-        "ledgerwallet==0.1.3",
+        "open-aea-ledger-ethereum>=1.51.0,<2",
+        "ledgerwallet~=0.1.3",
```

Comment:
Ledgerwallet is poorly maintained package atm, that's why it was pinned to a specific version other versions might lead to incompatibility issues

Reply:
Good to know, reverted 👌